Traceback (most recent call last):
  File "/home/thomasl/mambaforge/envs/py311/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/thomasl/mambaforge/envs/py311/lib/python3.11/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/thomasl/mambaforge/envs/py311/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/thomasl/mambaforge/envs/py311/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/thomasl/mambaforge/envs/py311/lib/python3.11/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/thomasl/mambaforge/envs/py311/lib/python3.11/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/thomasl/mambaforge/envs/py311/lib/python3.11/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import sys, os
from glob import glob
from importlib import reload


# modules in the L2 Sea Ice Concentration ATBD (v2) contain software code that implement the SIC algorithm
from sirrdp import rrdp_file
from pmr_sic import tiepoints as tp
from pmr_sic import hybrid_algo

# modules to read CIMR L1B files (for the dynamic tuning against an archive of past L1B files)
if '/Tools/' not in sys.path:
    sys.path.insert(0, os.path.abspath('../..') + '/Tools/')
from tools import io_handler as io
from tools import collocation as coll

import numpy as np
import json

# encoder class for writing JSON files
class MyEncoder(json.JSONEncoder):
    # https://stackoverflow.com/questions/27050108/convert-numpy-type-to-python/27050186#27050186
    def default(self, obj):
        if isinstance(obj, int):
            return int(obj)
        elif isinstance(obj, float):
            return float(obj)
        elif isinstance(obj, np.ndarray):
            return obj.tolist()
        else:
            return super(MyEncoder, self).default(obj)
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[1], line 7[0m
[1;32m      3[0m [38;5;28;01mfrom[39;00m [38;5;21;01mimportlib[39;00m [38;5;28;01mimport[39;00m reload
[1;32m      6[0m [38;5;66;03m# modules in the L2 Sea Ice Concentration ATBD (v2) contain software code that implement the SIC algorithm[39;00m
[0;32m----> 7[0m [38;5;28;01mfrom[39;00m [38;5;21;01msirrdp[39;00m [38;5;28;01mimport[39;00m rrdp_file
[1;32m      8[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpmr_sic[39;00m [38;5;28;01mimport[39;00m tiepoints [38;5;28;01mas[39;00m tp
[1;32m      9[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpmr_sic[39;00m [38;5;28;01mimport[39;00m hybrid_algo

[0;31mModuleNotFoundError[0m: No module named 'sirrdp'

